python/ray/dashboard/dashboard.py

```diff
@@ -786,6 +788,13 @@ def collect(self):
             if len(df) == 0:
                 continue
 
+            # start TensorBoard server if not started yet
+            if not self._tensor_board_started:
+                tb = program.TensorBoard()
```

Comment:
What happen if port 6006 has been bounded

Reply:
If port 6006 is already in use then the tensorboard will fail to start.